### PR TITLE
Initial spec with the main protocol and VTX

### PR DESCRIPTION
**Pull request by fiam**
Just a 1st draft, everything is up for discussion.

### CODE REVIEW

File: dan-proto-vtx-v1.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+- Input: None    
+- Output: *struct* DAN-VTX-CAPABILITIES-PAYLOAD-V2
+    - *uint16_t* Capabilities: Bitmask. See DAN-VTX-CAPABILITIES-MASK.
+    - *array[struct]* Bands
+        - *string* Name: Long band name (e.g. "BOSCAM A")
+        - *string* Short Name: Short name (usually 1 character long) for
+        showing the band in small indicators or next to a channel number
+        (e.g. "A").
+        - array[uint16_t]* Channels: Channel frequencies in MHz. See
```
Comment (DieHertz) [inline dan-proto-vtx-v1.md]: `*` missing

Comment (DieHertz) [inline dan-proto-vtx-v1.md]: `*` missing

File: dan-proto-v1.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ bus. Multiple devices of the same type are supported by using multiple uarts.
 
 ## Hardware requirements
 
-- DAN peripherals must use a baudrate of 115200, 1 stop bit and no parity bits.
+- DAN peripherals MUST initialy use a baudrate of 9600, 1 stop bit and no
```
Comment (mikeller) [inline dan-proto-v1.md]: 'initially'

**Comment by DieHertz** [comment]
We should demand every DAN device to be firmware-upgradable, no locked memory allowed

**Review by mikeller — APPROVED** [review]

**Review by DieHertz — APPROVED** [review]

**Comment by DieHertz** [comment]
A manufacturer asks for a 9600 baud option, I will shift the bitmask and start at 9600, but not lower than that.

**Comment by fiam** [comment]
There's no need to change the spec. The initial and lowest speed is 9600bps, so not declaring any additional bitrates will keep the peripheral at 9600bps.

**Pull request closed without merging**